### ISSUE TITLE
Add python demo forwarding all incoming SMS to specific email

### ISSUE THREAD

**Issue by QuentinCG** [QuentinCG/GSM-TC35-Python-Library#7]

**Comment by QuentinCG** [comment]
No need to add example to forward to a specific phone number since it is already easy from the library (if SIM allows it):
```python
from GSMTC35 import GSMTC35

gsm = GSMTC35()

if gsm.setup(_port="COM3", _pin="1234"):
  print("Enable SMS forwarding: "+str(gsm.setForwardStatus(GSMTC35.eForwardReason.UNCONDITIONAL, GSMTC35.eForwardClass.SMS, True, "+33601020304")))
```


Still to do: 
Add python demo forwarding all incoming SMS to specific email

**Comment by QuentinCG** [comment]
Example will be merged with #16 using this kind of API request:
`UPDATE http://127.0.0.1:8080/api/email with payload {'email': 'test@test.com', ...}`

Closing this issue since it is a duplicate.
